src/main/java/edu/ksu/canvas/attendance/controller/StudentSummaryController.java

```diff
@@ -35,9 +35,9 @@
 @Controller
 @Scope("session")
 @RequestMapping("/studentSummary")
-public class SummaryController extends AttendanceBaseController {
+public class StudentSummaryController extends AttendanceBaseController {
 
-    private static final Logger LOG = Logger.getLogger(SummaryController.class);
+    private static final Logger LOG = Logger.getLogger(StudentSummaryController.class);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "LOG" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1068)